docs/source/markdown/podman-secret-ls.1.md

```diff
@@ -16,6 +16,10 @@ Lists all the secrets that exist. The output can be formatted to a Go template u
 
 Format secret output using Go template.
 
+#### **\-\-noheading**
+
+Omit the table headings from the listing of secrets	.
```

Comment:
You have some kind of rogue white space, maybe a tab after the sentence and before the period.
```suggestion
Omit the table headings from the listing of secrets.
```